dev_tools/notebooks/notebook_test.py

```diff
@@ -73,8 +73,11 @@ def require_packages_not_changed():
 def env_with_temporary_pip_target():
     """Setup system environment that tells pip to install packages to a temporary directory."""
     with tempfile.TemporaryDirectory(suffix='-notebook-site-packages') as tmpdirname:
+        # Note: We need to append the PYTHONPATH here so that development Cirq,
```

Comment:
nit: s/append/prepend/

Reply:
Ack.  This test is run with `check/pytest` which sets PYTHONPATH to use the work-tree sources.
The line below appends `tmpdirname` to the PYTHONPATH.
When some notebook executes `pip install cirq`, that stable version would appear in PYTHONPATH after the worktree sources (which we want to use in this test).

Let me reword for clarity.

https://github.com/quantumlib/Cirq/blob/cc765a86064e7272047c427856ce89ba83953137/dev_tools/notebooks/notebook_test.py#L76-L83